integration-tests/tests/storage/document-collection.spec.ts

```diff
@@ -0,0 +1,487 @@
+/* eslint-disable no-process-env */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'reflect-metadata';
+import { ProjectType } from 'testkit/gql/graphql';
+import { initSeed } from 'testkit/seed';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createStorage } from '@hive/storage';
```

Comment:
if i add it to dependencies, it complains. if I add it to devDependencies it complains. So I ignored this.